test/urdfs/holonomic_base.urdf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+
+<robot name="hsrb">    <!--TODO: UnknownBodyException: body b 'hsrb' unknown     when robot name not "hsrb". Why??-->    
```

Comment:
when you merge urdf's with your merger, you attach the 2. urdf to the 1. one. So the result urdf has the name of the 1. urdf which is the holonomic base. So just change the order in the launch file